src/utils/line.ts

```diff
@@ -189,6 +194,7 @@ export const formatSecuredLineData = (
     positionIds: string[];
     positions: PositionMap;
   };
+  creditEvents: CreditEvent[];
   spigot: { type: CollateralTypes; line: string; tokenRevenue: { [key: string]: string } };
   escrow: {
     type: CollateralTypes;
```

Comment:
Can u add collateral events to escrow and spigot return data  separately please. 
And then in state when returning collateral events can you add the events specifically to the module address so 
`collateralMap[spigotAddress] = [...collateralMap[spigotAddress], ...linePageData.spigot.events]`

Reply:
@kibagateaux Can you please clarify what this means? 
`Can u add collateral events to escrow and spigot return data separately please.`

I'm not sure if there's a typo or something, but having a hard time understanding.

Reply:
Are you saying you want `collateralEvents` array within BOTH the `spigot` and `escrow` return objects? With the `spigot` return object, `collateralEvents` would represent revenue collateral events, and within the `escrow` return object, `collateralEvents` would represent liquid (i.e. token) collateral events?

Reply:
right now its one big `collateralEvents` array at the top level. This is (maybe) fine for the end SecuredLine type but we save the events in state to specific contracts that emit them so we need to separate collateral events if they are from Spigot or Escrow and add them to state under the corresponding address.

Reply:
![image](https://user-images.githubusercontent.com/11972548/207945931-75b497c1-1df3-4929-8171-d0acf837698f.png)

Reply:
removed `collateralEvents` array from top level. Added `events` array to `escrow` and adding to `spigot`